pkg/network/driver_interface.go

```diff
@@ -399,6 +512,34 @@ func createFlowHandleFilters() (filters []C.struct__filterDefinition, err error)
 	return filters, nil
 }
 
+func createDNSFilters() ([]C.struct__filterDefinition, error) {
+
+	var filters []C.struct__filterDefinition
+
+	ifaces, err := net.Interfaces()
```

Comment:
If an interface is created after system-probe starts we will not detect it. Do we need to periodically check this list of valid interfaces against what we have passed to the driver?

Reply:
The rest of the windows tracer assumes  a static list of interfaces. We should fix that, but I don't think it needs to be done in this change.